<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditKeyDialog</class>
 <widget class="QDialog" name="EditKeyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>  更改密码  </string>
  </property>
  <property name="windowIcon">
   <iconset resource="rc.qrc">
    <normaloff>:/new/prefix1/rc/actions/kgpg_sign_kgpg.png</normaloff>:/new/prefix1/rc/actions/kgpg_sign_kgpg.png</iconset>
  </property>
  <widget class="QGroupBox" name="GroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>更改密码</string>
   </property>
   <widget class="QLabel" name="oldkeylabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>旧 密 码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="newkeylabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>新 密 码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="surelabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>确认密码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="oldKeyLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="newKeyLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="sureKeyLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>89</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/new/prefix1/rc/actions/editdelete.png</normaloff>:/new/prefix1/rc/actions/editdelete.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/new/prefix1/rc/actions/apply.png</normaloff>:/new/prefix1/rc/actions/apply.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
